docs/configure-harvest-basic.md

```diff
@@ -347,29 +347,44 @@ authToken: $authToken
 EOF
 ```
 
-`get_credentials` that outputs authToken in YAML format for interacting with Keycloak auth provider:
-```bash
-#!/bin/sh
-curl "http://{KEYCLOAK_IP:PORT}/realms/{REAlM_NAME}/protocol/openid-connect/token" \
-  -X POST \
-  --header "Content-Type: application/x-www-form-urlencoded" \
-  --data-urlencode "grant_type=password" \
-  --data-urlencode "username={USERNAME}" \
-  --data-urlencode "password={PASSWORD}" \
-  --data-urlencode "client_id={CLIENT_ID}" \
-  --data-urlencode "client_secret={CLIENT_SECRET}" | jq | grep access_token | tr -d '"' | tr -d ',' | sed 's/access_token/authToken/'
-```
+Below are a couple of examples of how to create a credentials script that uses an OAuth2 workflow to authenticate with ONTAP or StorageGRID OAuth-enabled clusters.
```

Comment:
Below are a couple of OAuth2 credential script examples for authenticating with ONTAP or StorageGRID OAuth2-enabled clusters.

Reply:
Done